training/pytorch/structured/python_package/scripts/train-cloud.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
Please add a train-local.sh for the testing job to pick up, something like this: https://github.com/GoogleCloudPlatform/ai-platform-samples/blob/master/training/tensorflow/structured/base/scripts/train-local.sh

Reply:
Added a train-local.sh script that runs the training module directly.